Formula/procs.rb

```diff
@@ -17,16 +17,16 @@ class Procs < Formula
   def install
     system "cargo", "install", *std_cargo_args
 
-    system "#{bin}/procs", "--completion", "bash"
-    system "#{bin}/procs", "--completion", "fish"
-    system "#{bin}/procs", "--completion", "zsh"
-    bash_completion.install "procs.bash" => "procs"
-    fish_completion.install "procs.fish"
-    zsh_completion.install "_procs"
```

Comment:
I think these don't need `safe_popen_read`. There's no redirection going on, so I suspect calling `procs --completion bash` writes it to a file that's later installed. Instead, we can just remove the unnecessary `#{bin}` interpolation.